pep-0723.rst

```diff
@@ -0,0 +1,540 @@
+PEP: 723
+Title: Embedding pyproject metadata in single-file scripts
+Author: Ofek Lev <ofekmeister@gmail.com>
+PEP-Delegate: Brett Cannon <brett@python.org>
+Discussions-To: https://discuss.python.org/t/30979
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 04-Aug-2023
+Post-History: `04-Aug-2023 <https://discuss.python.org/t/30979>`__
+
+
+Abstract
+========
+
+This PEP specifies a format for defining metadata about single-file Python
+scripts, such as their runtime dependencies.
```

Comment:
This would read more cleanly (to me) along the lines of: "This PEP specifies a metadata format that can be embedded in single-file Python scripts to assist launchers, IDEs and other external tools which may need to interact with such scripts."

Reply:
The new text from after your comment is:

> This PEP specifies a format for defining metadata about single-file Python scripts that is required for proper runtime execution.

Is that satisfactory?

Reply:
I like your wording more so I just changed it, thanks!

Reply:
Better, yes.  I think it's the word "about" that makes it feel awkward, it triggers my brain to think something external, when that's not the intent - but it is well explained soon afterwards so there's not really a problem.